px-data-grid.html

```diff
@@ -239,6 +263,11 @@
              */
             remoteDataProvider: {
               type: Function
+            },
+
+            loading: {
```

Comment:
- documentation is missing
- should be read-only
- test is missing

Reply:
Done.